pkgs/development/python-modules/eth-hash/default.nix

```diff
@@ -31,7 +32,7 @@ buildPythonPackage rec {
 
   passthru.optional-dependencies = {
     pycryptodome = [ pycryptodome ];
-    pysha3 = [ pysha3 ];
+    pysha3 = if pythonOlder "3.9" then [ pysha3 ] else [ safe-pysha3 ];
```

Comment:
We only really support python 3.10 and 3.11 right now, so we could also drop pysha3 in the future.

Reply:
#232845